pom.xml

```
@@ -116,6 +116,8 @@
     <scmTag>HEAD</scmTag>
     <!-- Where to put temporary files during test runs. -->
     <surefireTempDir>${project.build.directory}/tmp</surefireTempDir>
+    <spotless.version>2.22.1</spotless.version>
+    <sort.nrOfIndentSpace>2</sort.nrOfIndentSpace>
```

```suggestion
    <sortPom.nrOfIndentSpace>2</sort.nrOfIndentSpace>
```

Maybe? (+ downstream changes)

Or even

```suggestion
    <spotless.sortPom.nrOfIndentSpace>2</sort.nrOfIndentSpace>
```

to make it really obvious what this is related to.

---

I am fine with

```suggestion
    <spotless.sortPom.nrOfIndentSpace>2</spotless.sortPom.nrOfIndentSpace>
```